### ISSUE TITLE
Question on inputting as PDB/structure files rather than .fasta for multimer predictions

### ISSUE THREAD

**Issue by wentski** [deepmind/alphafold#279]
Hello,

We have been using Alphafold in order to predict protein:protein interactions using the multimer function. In this case, quite a lot of the processing time is taken up by folding a protein sequence for which we already know the structure. As we are more interested in protein complex formations and interactions, is there any way to input PDB files (or similar) rather than .fasta files and effectively bypass the protein folding steps?

**Comment by NickEdmunds** [comment]
This was also my question to AlphaFold: Could I ask if its possible to feed pdb files, i.e. model structures straight into alphafold multimer? This seems to be implied in your paper where Alphafold and cluspro structures are compared. If this is possible, could you give me some indication of how it is achieved.

**Comment by smouilleron** [comment]
I totally agree with the previous comment, it would be great to be able to do this for multimer interactions... Or at least to use DeepMind pdb database of 200millions proteins instead of refolding it each time... To be able to input a known A-B complex and search for how C is binding would also be great.
